fiftyone/utils/hrm2.py

```
@@ -0,0 +1,2550 @@
+    """Per-person HRM2 prediction metadata.
+
+    This stores all HRM2 output data for a single detected person,
+    including SMPL parameters, camera, and 3D keypoints. This is a
+    dynamic document that can hold any additional attributes.
+
+    Standard attributes set by HRM2:
+        person_id: unique identifier for this person in the frame
+        smpl_params: SMPLParams document with body model parameters
+        camera_weak_perspective: [scale, tx, ty] weak perspective camera
+        camera_translation: [tx, ty, tz] full camera translation
+        keypoints_3d: list of 3D keypoint coordinates (25 × 3 for BODY-25)
```

Ruff (0.14.6)

Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)